lib/vaahextendflutter/services/storage/local/flutter_secure_storage/flutter_secure_storage_impl.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
+
+import '../../storage.dart';
+
+class FlutterSecureStorageImpl implements Storage {
+  final _storage = const FlutterSecureStorage();
+
+  @override
+  Future<void> init() async {}
+
+  @override
+  Future<void> create({required String key, required String value}) async {
+    await _storage.write(key: key, value: value);
+  }
+
+  @override
+  Future<void> createAll({required Map<String, String> values}) async {
+    for (String k in values.keys) {
+      await _storage.write(key: k, value: values[k]);
+    }
+  }
```

Comment:
@themodernpk sir,

should we have one method create only like below or two?
```dart
  Future<void> create({required Map<String, String> values}) async {
```

or
```dart
  Future<void> create({required String key, required String value}) async {
  //
  Future<void> createAll({required Map<String, String> values}) async {
```

Reply:
same for other methods, Also update operation is missing

Reply:
@we-prajapati-c001 we should have the following methods:
```
create
createMany // it will accept array
updateOrCreate // it will try to find the key, if the key exists then it will update otherwise it will create
read
readMany
readAll
update
updateMany
delete
deleteMany
deleteAll

```

Reply:
Thank you sir!

Reply:
The `update`  and `create` operation is done by the same method in both Hive and FlutterSecureStorage.
Therefore, instead of creating new methods, I think changing the name from `create` to `save` should convey the message that it also updates the values and the class will not repeat the same code with different method names.

**From**
```dart
Future<void> create({required String key, required String value});

Future<void> createAll({required Map<String, String> values});
```
**To**
```dart
Future<void> save({required String key, required String value});

Future<void> saveMany({required Map<String, String> values});
```

The users will use it like below example:
```dart
storage.save(key: 'key1', value: 'value1');

storage.saveMany(values: {
   'key1': 'Value1',
   'key2': 'Value2',
   'key3': 'Value3',     
   'key4': 'Value4',
   'key5': 'Value5',
  },
);

Reply:
I suggest keeping it as `create`. Let's stick with `create` for clarity and simplicity. Because if we are using `save` then you have to rename other's as well for update - `set`, for read - `find` and for delete - `remove`. Either keep `save, find, set, remove` or `create read update delete`. Please go with Pradeep sir suggestion.

Reply:
https://github.com/webreinvent/vaahflutter/pull/43#discussion_r1630711751